zola.metainfo.xml

```diff
@@ -52,6 +52,9 @@
   <content_rating type="oars-1.1" />
 
   <releases>
+    <release version="0.16.0" date="2022-07-16">
```

Comment:
Instead of doing this, please use the upstream one (https://github.com/getzola/zola/blob/master/zola.metainfo.xml). However, you'll need to patch it as 0.16.0 wasn't added to it. Let me know if you need any help with this.

Reply:
So I should:

1. Remove this repos `metainfo.xml`
2. Add a patch that corrects the tarballs version numbers and dates which will need to be removed next update or changed every update

Reply:
> Remove this repos metainfo.xml

Yes.

> Add a patch that corrects the tarballs version numbers and dates which will need to be removed next update or changed every update

Yes, and also make a pull request to the upstream Zola repo to fix the metainfo there. This is what we agreed upon with upstream when we put the metainfo there.